.ado/pipelines/templates/stages-loadtest-azure.yaml

```diff
@@ -56,7 +56,16 @@ stages:
     - template: steps-parse-terraform-output.yaml
       parameters:
         workingDirectory: '${{ parameters.terraformWorkingDirectory }}'
-        displayName: 'Parse MALT deployment outputs'
+        displayName: 'Parse loadtest deployment outputs'
+
+    # Set loadTestBaseline variable
+    - task: PowerShell@2
+      displayName: 'Set load test baseline file'
+      inputs:
+        targetType: inline
+        script: |
+          echo "*** Setting loadTestBaseline to $(azureLoadTestBaseline)"
+          echo "##vso[task.setvariable variable=loadTestBaseline]$(azureLoadTestBaseline)"
```

Comment:
why do you need to set the variable? it already is one just with a different name?!

Reply:
> just to double check: the actual fix is to move to UK, right?

I moved e2e (back) to `uksouth` as chaos is still not available in `swedencentral`. The issue/fix itself is the changed ordering of the steps, moving "setting loadTestBaseline" outside of the `${{ if eq(parameters.embeddedLoadTest, 'true') }}:` expression.

`loadTestBaseline` is used as the input for `loadtest-create.ps1` - in the `configuration.yaml` we specify two options either `azureLoadTestBaseline` or `chaosLoadTestBaseline` and the pipeline defines, based on some expressions which one will be used for `loadTestBaseline `

Reply:
Default is:

https://github.com/Azure/Mission-Critical-Online/blob/a5552ca1f8324ca7659b81754b26efc043550d4f/.ado/pipelines/templates/stages-loadtest-azure.yaml#L61-L68

And when `runChaosTesting` is set to `true` it will be set to:

https://github.com/Azure/Mission-Critical-Online/blob/a5552ca1f8324ca7659b81754b26efc043550d4f/.ado/pipelines/templates/stages-loadtest-azure.yaml#L88-L96

Reply:
ah got it, thanks